modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/java/com/liferay/frontend/data/set/views/web/internal/portlet/FDSViewsPortlet.java

```diff
@@ -471,6 +471,47 @@ private void _createFDSFieldObjectDefinition(
 			ObjectRelationshipConstants.TYPE_ONE_TO_MANY, null);
 	}
 
+	private void _createFDSCardsSectionObjectDefinition(
```

Comment:
This code is technically perfectly fine, but Brian would not allow a private method that is unused. You can remove this, and I'll wire backend in a followup, or you can soldier through and add missing bits:

1. This needs to be [invoked](https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/java/com/liferay/frontend/data/set/views/web/internal/portlet/FDSViewsPortlet.java#L672-L675), behind FF, along with `FDSListSection`.
1. New [modifiable system object](https://github.com/liferay/liferay-portal/blob/master/modules/apps/object/object-api/src/main/java/com/liferay/object/definition/util/ObjectDefinitionUtil.java#L122). This also defines a new endpoint alias, `/data-set-manager/cards-sections`
1. New endpoint in [constants](https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/utils/constants.ts#L14) , and using it as `API_URL.FDS_CARDS_SECTIONS`

Reply:
I'm gonna try to finish everything :smile: 

Thanks!

Reply:
It works now :heart_eyes: